packages/example/pages/_app.tsx

```diff
@@ -98,41 +100,57 @@ const { wallets } = getDefaultWallets({
   projectId,
 });
 
-const connectors = connectorsForWallets([
-  ...wallets,
-  {
-    groupName: 'Other',
-    wallets: [
-      argentWallet({ chains, projectId }),
-      bifrostWallet({ chains, projectId }),
-      bitKeepWallet({ chains, projectId }),
-      bitskiWallet({ chains }),
-      coin98Wallet({ chains, projectId }),
-      coreWallet({ chains, projectId }),
-      dawnWallet({ chains }),
-      enkryptWallet({ chains }),
-      foxWallet({ chains, projectId }),
-      frameWallet({ chains }),
-      frontierWallet({ chains, projectId }),
-      imTokenWallet({ chains, projectId }),
-      ledgerWallet({ chains, projectId }),
-      mewWallet({ chains }),
-      okxWallet({ chains, projectId }),
-      omniWallet({ chains, projectId }),
-      oneKeyWallet({ chains }),
-      phantomWallet({ chains }),
-      rabbyWallet({ chains }),
-      safeheronWallet({ chains }),
-      tahoWallet({ chains }),
-      talismanWallet({ chains }),
-      tokenPocketWallet({ chains, projectId }),
-      trustWallet({ chains, projectId }),
-      uniswapWallet({ chains, projectId }),
-      xdefiWallet({ chains }),
-      zerionWallet({ chains, projectId }),
-    ],
-  },
-]);
+// In this example, the user's locale is stored in local storage, allowing for language switching functionality.
+// If defined, we retrieve the locale value and typecast it as `Language`.
+export const localStorageLocale =
+  typeof localStorage !== 'undefined'
+    ? (localStorage.getItem('locale') as Language)
+    : null;
```

Comment:
I think we can drop the need for localstorage in this example; we want to show passing down the locale to the RainbowKitProvider prop tree. dApps can implement that however they want, but we should consider:
1) the router locale
2) checking `navigator.language` for the user's preferred language (I don't think that is handled by the above?)
3) Altering the drop-down selection
The drop-down default can be set by 1 and 2, and we keep track of the current selection in useContext or useState; this will be a better pattern to demonstrate to developers

Reply:
Yeah let's get rid of `localStorage` and focus on drop down and state managment.

All things added including `navigator.language` ✨